roles/kubernetes/kubeadm/defaults/main.yml

```diff
@@ -9,7 +9,7 @@ kubeadm_use_file_discovery: "{{ remove_anonymous_access }}"
 
 # If non-empty, will use this string as identification instead of the actual hostname
 kube_override_hostname: >-
-  {%- if cloud_provider is defined and cloud_provider in ['aws'] -%}
+  {%- if cloud_provider is defined -%}
```

Comment:
Hum, this looks like an aws specific tweaks, which was not applied to `external`, so we should probably get rid of that condition altogether instead.